clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -313,6 +321,10 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
       description.events << new CreateServerGroupEvent(
         AmazonCloudProvider.ID, account.accountId, region, asgName
       )
+
+      deploymentResult.deployments.add(
+          new DeploymentResult.Deployment(location: region, serverGroupName: asgName, capacity: capacity)
```

Comment:
neato! But shouldn't we add the cloud provider and account here for completeness?

Reply:
That won't hurt ... consider it done!